i18n-helpers/src/bin/markdown-gettext.rs

```diff
@@ -0,0 +1,248 @@
+use anyhow::{anyhow, bail, Context};
+use mdbook_i18n_helpers::translate;
+use polib::catalog::Catalog;
+use polib::po_file;
+use std::fs::File;
+use std::io::{BufReader, Read, Write};
+use std::path::PathBuf;
+use std::{env, fs};
+
+fn split_argument(arg: &String) -> Option<String> {
+    let splitted: Vec<_> = arg.split('=').collect();
+    if splitted.len() == 2 {
+        return Some(splitted[1].to_string());
+    }
+
+    None
+}
+
+fn build_catalog(lang_option: Option<String>) -> anyhow::Result<Catalog> {
+    match lang_option {
+        Some(lang_path) => {
+            let pot_path = PathBuf::from(&lang_path);
+
+            if pot_path.exists() {
+                let catalog = po_file::parse(&pot_path)
+                    .map_err(|err| anyhow!("{err}"))
+                    .with_context(|| format!("Could not parse {:?} as PO file", pot_path))?;
+
+                return Ok(catalog);
+            }
+
+            bail!("--po must be specified")
+        }
+        None => bail!("--po must be specified"),
```

Comment:
You should be able to remove the `match` with

```rust
let lang_option = lang_option.ok_or(anyhow!("--po must be specified"))?;
```

This will return early if the option is empty. See [`ok_or`](https://doc.rust-lang.org/stable/std/option/enum.Option.html#method.ok_or) and [`anyhow!`](https://docs.rs/anyhow/latest/anyhow/macro.anyhow.html).